sailor/assetcentral/notification.py

```diff
@@ -294,6 +303,7 @@ def create_notification(**kwargs) -> Notification:
 
     Example
     -------
+    ?? Why is this example with ">>>" and "..." ??
```

Comment:
looks like it was copied from a python session :)
should be removed, same in `update_notification` (good find!)